install_base.sh

```diff
@@ -314,6 +314,8 @@ for arg in "$@"; do
         # gettext for autopoint
         pacman_packages+=(gettext autoconf libtool bison)
 
+        git clone https://github.com/alpinelinux/alpine-chroot-install.git --depth=1 -C ${LISA_HOME}/tools
```

Comment:
Just realized this needs to be split in a separate function, otherwise the repo could be cloned twice depending on the command line invocations. See how `install_nodejs_snap` is added to the list of functions to call and to the ordered list.